R/dockerfile.R

```diff
@@ -1,24 +1,16 @@
-# Copyright 2016 Daniel Nüst
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
+# Copyright 2016 Opening Reproducible Research (http://o2r.info)
 
 #' dockerfile
 #'
 #' Create a Dockerfile from the
 #'
 #' @param from the source of the information to construct the Dockerfile
-#' @param to path and file name to save the Dockerfile to
+#' @param env the environment that should be included in the image
```

Comment:
Please clarify the difference between `env` and `context`. `env` holds environment variables, so this should probably be a named list, right?

Please add links to the Dockerfile spec [e.g. this one here](https://docs.docker.com/engine/reference/builder/#/env) where that is shorter than explaining yourself.